src/common/database/commands/DeleteNodeCommand.ts

```diff
@@ -0,0 +1,21 @@
+import { DatabaseCommand } from "../DatabaseCommand";
+import { QueryConfig, QueryResult } from "pg";
+import { DatabaseManager } from "../DatabaseManager";
+
+export class DeleteNodeCommand extends DatabaseCommand<void> {
+
+    public constructor(private readonly id: string, private readonly table: string) {
+        super();
+    }
+
+    public getQueryConfig(): QueryConfig<any[]> {
+        return {
+            text: `DELETE FROM ${this.table} WHERE id=$1`,
+            values: [this.id]
+        };
+    }
+    public setDatabaseResult(databaseManager: DatabaseManager, result: QueryResult<any>): DatabaseCommand<any>[] {
```

Comment:
it could be that the node is already deleted so no column is affected
or the node exists, and one column is effected, because id is for node tables always unique
so we could do that, however I don't see any benefits because both possible results are correct

Reply:
Ok, for deleting thats an ok solution